job_templates/snippet/publisher_cobertura.xml.em

```diff
@@ -1,5 +1,5 @@
     <hudson.plugins.cobertura.CoberturaPublisher plugin="cobertura@@1.14">
-      <coberturaReportFile>ws/build*/**/*coverage.xml</coberturaReportFile>
+      <coberturaReportFile>ws/build/*coverage.xml</coberturaReportFile>
```

Comment:
These looks incorrect. Why is the `*` removed from the `build` directory name?

Reply:
> These looks incorrect. Why is the `*` removed from the `build` directory name?

is there any option in CI which modified the [ws/build/ in the workspace](https://ci.ros2.org/view/All/job/test_linux_coverage/ws/ws/) or other reason to keep the wildcard on the build directory name? Not sure why was there before.

Reply:
Yes, there is a reason to keep the wildcard: https://github.com/ros2/ci/blob/dea332718366fe745f4eabab735bb0db9d65a450/ros2_batch_job/__main__.py#L187-L190

Reply:
Thanks for the context: reverted aafe41e53164f49cd7b06abec318bbeb723ceb6f